src/clr-core/.storybook/public/demo.css

```diff
@@ -142,10 +142,16 @@ cds-card {
   box-shadow: 0 -0.05rem 0 hsl(198, 0%, 70%) inset;
 }
 
-.demo-alt-navicon {
+.demo-alt-header cds-icon,
+.demo-alt-header-2 cds-icon {
   opacity: 0.6;
 }
 
+.demo-alt-header cds-icon:first-child,
+.demo-alt-header-2 cds-icon:first-child {
+  opacity: 1;
+}
+
```

Comment:
Minor clean up, help remove the class from the code example demo so it focus more on the layout APIs